tests/testthat/test-utils.R

```diff
@@ -13,3 +13,36 @@ test_that("rbindlist", {
   expect_equal(data.frame(), rbindlist(character()))
 
 })
+
+
+test_that("dims to col & row and back", {
+
+  exp <- list(c("A", "B"), c("1", "2"))
+  got <- dims_to_rowcol("A1:B2")
+  expect_equal(exp, got)
+
+  exp <- list(1:2, 1:2)
+  got <- dims_to_rowcol("A1:B2", as_integer = TRUE)
+  expect_equal(exp, got)
+
+  exp <- list(1:2, c(1L))
+  got <- dims_to_rowcol("A:B", as_integer = TRUE)
+  expect_equal(exp, got)
+
+  exp <- list("A", c("1"))
+  got <- dims_to_rowcol("A:A", as_integer = FALSE)
```

Comment:
If we ever use this to create the rows to write to, rows should be something else. Maybe return `Inf` or something. In Excel logic this will use as many rows as are available.